src/components/FeaturesDemo/constants.ts

```diff
@@ -0,0 +1,164 @@
+export const PAUSE_XS = 500;
+export const PAUSE_SM = 1000;
+export const PAUSE_MD = 2500;
+export const PAUSE_LG = 5000;
+
+export const getById = <T extends { id: number | string }>(items: T[], id: number | string): T => {
+    const result = items.find(item => item.id === id);
+
+    if (!result) {
+        throw new Error(
+            "Couldn't find item with id " + id + " in collection " + JSON.stringify(items),
+        );
+    }
+
+    return result;
+};
+
+export enum HeadingId {
+    FamiliarApi,
+    ReplMode,
+    HtmlReport,
+    GuiMode,
+    VisualTesting,
+}
+
+export const HEADINGS = [
+    { id: HeadingId.FamiliarApi, data: "Familiar and concise API, that works everywhere" },
+    { id: HeadingId.ReplMode, data: "Get instant feedback as you write tests" },
+    { id: HeadingId.HtmlReport, data: "View data results in feature-rich report" },
+    { id: HeadingId.GuiMode, data: "Powerful, yet easy to use GUI mode" },
+    { id: HeadingId.VisualTesting, data: "Vast visual testing capabilities" },
+];
+
+export const LONGEST_HEADING = HEADINGS.reduce((a, b) =>
+    a.data.length > b.data.length ? a : b,
+).data;
+
+export enum CodeId {
+    BasicTestBegin,
+    BasicTestEnd,
+    ReplTestBegin,
+    BrowserUrl,
+    FindInput,
+    ScrollToInput,
+    SetValue,
+    ButtonClick,
+    Expect,
+    ReplTestEnd,
+}
+
+export const CODE_MESSAGES = [
+    { id: CodeId.BasicTestBegin, messages: [`describe("Main Store Page", () => {`] },
+    { id: CodeId.ReplTestBegin, messages: [`<br>  it("should add product to cart", () => {<br>`] },
+    {
+        id: CodeId.BrowserUrl,
+        messages: [`<span></span>    await browser.url("localhost:8000");<br>    <span></span>`],
+    },
+    {
+        id: CodeId.FindInput,
+        messages: [
+            `const input = await browser.$("HELLO");`,
+            `const input = await browser.$("input[data-testid='search']");<br>    <span></span>`,
+        ],
+    },
+    { id: CodeId.ScrollToInput, messages: [`await input.scrollIntoView();<br>    <span></span>`] },
+    {
+        id: CodeId.SetValue,
+        messages: [`await input.setValue("Company of one");<br>    <span></span>`],
+    },
+    {
+        id: CodeId.ButtonClick,
+        messages: [
+            `await browser.$("button[data-testid='add-to-cart']").click();<br><br>    <span></span>`,
+        ],
+    },
+    {
+        id: CodeId.Expect,
+        messages: [
+            `const cartCount = await browser.$("span[data-testid='cart-count']");<br>    await expect(cartCount).toHaveText('(1)');`,
+        ],
+    },
+    { id: CodeId.ReplTestEnd, messages: [`<br>  });<br>`] },
+    {
+        id: CodeId.BasicTestEnd,
+        messages: [
+            `<br>  it("should render product card", async ({browser}) => {
+    await browser.url("localhost:8000");
+
+    await browser.assertView("div[data-data-id='product']", "default-state");
```

Comment:
In this case first argument must be state and the second one selector.
But I suggest write it like this - `await browser.$("div[data-data-id='product']").assertView("default-state")`